File: Prashant Kumar Singh/day-8/8b.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int a = 0;  
        int i = numbers.size() - 1;

        while (numbers[a] + numbers[i] != target)
            if (numbers[a] + numbers[i] < target)
                ++a;
            else
                --i;

        return {a+1, i+1};
    }
};

File: Mahek Jhamb/Day 8/8B.py
class Solution:
    def twoSum(self, numbers: list[int], target: int) -> list[int]:
        l,r=0,len(numbers)-1
        while(l<r):
            currsum=numbers[l]+numbers[r]
            if(currsum==target):
                return [l+1,r+1]
            elif (currsum>target):
                r-=1
            else:
                l+=1
